src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -585,7 +585,7 @@ private static CultureData CreateCultureWithInvariantData()
             invariant._sNegativeSign = "-";                    // negative sign
             invariant._iDigits = 2;                      // number of fractional digits
             invariant._iNegativeNumber = 1;                      // negative number format
-            invariant._waGrouping = new int[] { 3 };          // grouping of digits
+            invariant._waGrouping = [3];          // grouping of digits
```

Comment:
I wonder if it'd be relevant to have a small cache of these single entry arrays that are kept internal.

We have a few of them, particularly here in globalization